analyzers/tests/SonarAnalyzer.UnitTest/TestCases/FieldShouldBeReadonly.CSharp11.cs

```diff
@@ -29,10 +29,10 @@ record PointerTypes
 
     private void AssignValue()
     {
-        _nint4 = 42;
+    _nint4 = 42;
 
-        _nint5++;
-        _nuint5--;
+    _nint5++;
+    _nuint5--;
```

Comment:
The indentation is still wrong,  the code in the body of the method should be indented by 4 more spaces than the opening and closing curly braces.